country-a-service/epps-service/src/main/java/dk/sundhedsdatastyrelsen/ncpeh/service/mapping/DispensationMapper.java

```diff
@@ -256,61 +265,41 @@ String authorRole(Document cda) throws XPathExpressionException {
         throw new IllegalArgumentException("Unexpected function code: " + functionCode);
     }
 
-    private boolean notBlank(String s) {
+    private static boolean notBlank(String s) {
         return s != null && !s.isBlank();
     }
 
-    /**
-     * TODO FMK Improvement
-     * FMK is supposed to support "Udenlandsk" OrganisationIdenfier, according to the documentation
-     * When called, it responds with an error that it is not supported yet
-     * <p>
-     * Two options going forward:
-     * - FMK start supporting Udenlandsk, and we reimplement forwarding the identifier from Country-B
-     * - We agree with FMK to "proxy" all requests through another Organisation, like we do in the code right now.
-     * <p>
-     * (2024/09/10)
-     */
-    private OrganisationIdentifierType identifier(Node id) {
-
-//        var attrs = id.getAttributes();
-//        var root = attrs.getNamedItem("root");
-//        var ext = attrs.getNamedItem("extension");
-//        if (root == null) {
-//            throw new IllegalArgumentException("Id nodes without root attributes are unsupported");
-//        }
-//        return OrganisationIdentifierType.builder()
-//            .withSource(OrganisationIdentifierPredefinedSourceType.UDENLANDSK.value())
-//            .withValue(ext == null
-//                ? root.getTextContent()
-//                : String.format("%s.%s", root.getTextContent(), ext.getTextContent()))
-//            .build();
-        return TestIdentities.skanderborgApotek;
+    private static OrganisationIdentifierType placeholdePharmacyId() {
```

Comment:
```suggestion
    private static OrganisationIdentifierType placeholderPharmacyId() {
```

But why is it called placeholder when it's the final one?

Also, we should probably use the same organisation identifier in the calls to the authorization registry, unless I'm mixing something up.

Reply:
The identities we use for system calls (like for the authorization registry) also has certificates.  So it's a different kind of issue.  We need to request certificates for NCPeH that we can use.